src/components/checkboxFilter/index.tsx

```diff
@@ -53,16 +55,35 @@ function CheckboxFilter<ItemKey extends string>({
             key={`filter_${name}_${item.label}`}
             name={`filter_${name}_${item.label}`}
             label={
-              <chakra.span
-                w="full"
-                display="flex"
-                justifyContent="space-between"
-              >
-                <chakra.span>{item.label}</chakra.span>
-                <chakra.span>
-                  {addThousandSeparatorToNumber(item.facet)}
+              item.image ? (
+                <chakra.div
+                  w="full"
+                  display="flex"
+                  alignItems="center"
+                  justifyContent="space-between"
+                >
+                  <chakra.div width="119px" height="80px">
```

Comment:
I don't believe that works for all our filters. The color icon is much much smaller. I would leave that up to the image that is passed